common/model-ratio.go

```diff
@@ -5,6 +5,19 @@ import (
 	"strings"
 )
 
+var DalleSizeRatios = map[string]map[string]float64{
+	"dall-e-2": {
+		"256x256":   1,
+		"512x512":   1.125,
+		"1024x1024": 1.25,
+	},
+	"dall-e-3": {
+		"1024x1024": 1,
+		"1024×1792": 2,
```

Comment:
`×` => `x`

Reply:
General x cross is acceptable
Simply copied from https://platform.openai.com/docs/guides/images/usage?context=node

Reply:
The actual issue here is not whether OpenAI accepts `×`, but rather the inability to pass the resolution verification in the subsequent code. [controller/relay-image.go#90](https://github.com/songquanpeng/one-api/pull/679/files#diff-1dd7b882b32072873f2c5cbc0ea2d69af42208f0c63c702dd3d662907725fe61R90)

Reply:
> `×` => `x`

My previous comment means to replace all occurrences of `×` with `x` or vice versa. This way, the relay can handle both `x` and `×` simultaneously. The current code fails to pass the `×` verification at 1024x1024 resolution.

Reply:
Get it, I will review this later 👍🏻

Reply:
It is my Windows VSC issue idk why